scripts/db/populate_departments.sql

```diff
@@ -127,7 +127,7 @@ INSERT INTO departments (dept_name, is_enrolled) VALUES
 
 WITH listings (dept_name, subject_area, catalog_id) AS (VALUES
   ('African American Studies', 'AFRICAM', NULL),
-  ('Agricultural and Resource Economics', 'A,RESEC', NULL),
+  ('Agricultural and Resource Economics', 'ARESEC', NULL),
```

Comment:
Here and below, we have to shift over to the Campus Solutions world of alpha-only subject areas.